lte/gateway/c/oai/tasks/mme_app/mme_app_main.c

```diff
@@ -484,7 +487,10 @@ static void* mme_app_thread(__attribute__((unused)) void* args) {
   send_app_health_to_service303(&mme_app_task_zmq_ctx, TASK_MME_APP, false);
   start_stats_timer();
 
-  zloop_start(mme_app_task_zmq_ctx.event_loop);
+  int zloop_mme_task_res = zloop_start(mme_app_task_zmq_ctx.event_loop);
+  zloop_set_verbose(mme_app_task_zmq_ctx.event_loop, true);
```

Comment:
this needs to be called before zloop_start; zloop_start doesn't return until something interrupts it.